src/api/api.js

```diff
@@ -1,17 +1,5 @@
 import { fetch } from './fetch.js'
 
-export function getInstitutions() {
```

Comment:
We no longer make a call for the list of institutions. We use the token data instead.